openedx/core/djangoapps/content_libraries/models.py

```diff
@@ -127,3 +202,248 @@ def save(self, *args, **kwargs):  # lint-amnesty, pylint: disable=arguments-diff
     def __str__(self):
         who = self.user.username if self.user else self.group.name
         return f"ContentLibraryPermission ({self.access_level} for {who})"
+
+
+class LtiProfileManager(models.Manager):
+    """
+    Custom manager of LtiProfile mode.
+    """
+
+    def get_from_claims(self, *, iss, aud, sub):
```

Comment:
Same here, I don't understand the purpose of the `, *,` especially as you are always calling this method using `kwargs` like `iss=x, aud=y`.

Reply:
Oh nice, TIL. I'm a fan of that, keyword arguments all the way.